templates/zerver/api/writing-bots.md

```diff
@@ -56,10 +56,10 @@ class MyBotHandler(object):
     A docstring documenting this bot.
     '''
 
-    def usage(self):
+    def usage(self) -> str:
         return '''Your description of the bot'''
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(self, message, bot_handler) -> None:
```

Comment:
This seems unhelpful -- it shows a function annotated only on the return type, which isn't something anyone should write.

If we add types to this example, let's have them follow the `handle_message` methods in our own python-zulip-api repo.

Reply:
I see, should the line be https://github.com/zulip/python-zulip-api/blob/ced3a97b3f0915536ba85a9515f8abdf96384e96/zulip_bots/zulip_bots/bots/helloworld/helloworld.py#L15 ? I `grep`-ed throughout python-zulip-api and only found one that has the Python 3 syntax.

Reply:
Try updating your python-zulip-api clone -- maybe they added a bunch in the past couple of weeks. I grepped for `def handle_message` and found a lot of examples, most of them like
```
    def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
```

Reply:
Right, I just rebased and saw the changes. Weighing on https://github.com/zulip/zulip/commit/ca09c1e0fde873264a319080aa996edfd1817cdf#commitcomment-26229756, what should be the path forward here?